src/test/scala/com/target/data_validator/validator/ColumnBasedSpec.scala

```diff
@@ -79,10 +87,49 @@ class ColumnBasedSpec extends FunSpec with Matchers with TestingSparkSession {
 
     it("should fail when numeric column doesn't match max value") {
       val dict = new VarSubstitution
-      val sut = mkValidatorConfig(List(ColumnMaxCheck("number", Json.fromInt(100)))) // scalastyle:ignore
+      val columnMaxCheck = ColumnMaxCheck("number", Json.fromInt(100))
+      val sut = mkValidatorConfig(List(columnMaxCheck)) // scalastyle:ignore
+      assert(!sut.configCheck(spark, dict))
+      assert(sut.quickChecks(spark, dict))
+      assert(sut.failed)
+      assert(columnMaxCheck.getEvents contains
+        ColumnBasedValidatorCheckEvent(true,
+          LinkedHashMap("expected" -> "100", "actual" -> "3", "error_percent" -> "97.00%").toMap,
+          "ColumnMaxCheck number[IntegerType]: Expected: 100, Actual: 3. Error %: 97.00%"))
```

Comment:
If you incorporate my suggestions above the formatting here and below will need updated.